package.json

```diff
@@ -52,7 +60,7 @@
   "lint-staged": {
     "src/**/*.{js,jsx,ts,tsx,json,css,scss,md}": [
       "prettier --write",
-      "eslint src/ --ext .js --ext .jsx --fix",
+      "eslint src/ --fix",
```

Comment:
During the rebase, you were supposed to maintain line 55 that tells `eslint` to lint both `.js` and `.jsx` files